### DISCUSSION TITLE
Esmerald is not a Starlette framework anymore

### DISCUSSION THREAD

**Discussion by hasansezertasan** [encode/starlette#2575] [Q&A] [answered]
As per the [README](https://github.com/dymmond/esmerald/blob/a0892f9a60d25fb4648e4f8119b66c1f5bc1ec99/README.md?plain=1#L38-L39) file of the package, the Esmerald doesn't use/depend on Starlette anymore.

So I'm not sure if it still fits to go under the "Frameworks" section in the "Third Party Packages" page.

https://github.com/encode/starlette/blob/96c90f26622c8f243ad965371eae2c2028a518de/docs/third-party-packages.md?plain=1#L171-L180

**Comment by Kludex** [comment, marked as answer]
👍 

https://github.com/encode/starlette/pull/2578

**Reply by Kludex** [reply]
It's always easier if you create the PR.
